nebula/ui/components/VPNMobileStatusBarModifier.qml

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+
+import Mozilla.VPN 1.0
+
+QtObject {
+    id: statusBarModifier
+
+    enum StatusBarTextColor {
+        Light,
+        Dark
+    }
+    property int statusBarTextColor: VPNMobileStatusBarModifier.StatusBarTextColor.Dark
+
+    function applyChanges() {
+        VPN.setStatusBarTextColor(statusBarModifier.statusBarTextColor);
+    }
+
+    function resetDefaults() {
+        VPN.setStatusBarTextColor(VPNMobileStatusBarModifier.StatusBarTextColor.Dark);
+    }
+
+    Component.onCompleted: applyChanges()
```

Comment:
Maybe we can set the status-bar color in the CTOR of theme.cpp. And reset it when the property changes.

Reply:
I implemented this component to overwrite whatever is the default status bar color. For the case described in the Jira ticket we want the text color to be light on onboarding pages and dark in other pages. That is why I chose to apply changes when the component is created and reset them when it's destroyed.

When there is a dark theme, it would be light in both and yes we would probably need to set that on the constructor of theme.cpp or main.qml. However, as I understand it, this is a different case.

Reply:
makes sense!